### PR TITLE
Charters views integration

### PR DESCRIPTION
**Pull request by entantoencuanto**
related with https://github.com/PopulateTools/issues/issues/441, https://github.com/PopulateTools/issues/issues/415 and https://github.com/PopulateTools/issues/issues/459

Closes https://github.com/PopulateTools/issues/issues/461, https://github.com/PopulateTools/issues/issues/460

## :v: What does this PR do?
This PR is on top of https://github.com/PopulateTools/gobierto/pull/1929. Do not merge before merging the other one.

This PR integrates the views defined in the sandbox

### Before this PR
screen shot 2018-10-17 at 19 13 02
screen shot 2018-10-17 at 22 25 56

### After this PR
screen shot 2018-10-17 at 19 12 01

[See in staging](http://burjassot.gobify.net/cartas-de-servicios/ayuntamiento-de-valencia)

screen shot 2018-10-17 at 23 00 39

[See in staging](http://burjassot.gobify.net/cartas-de-servicios/ayuntamiento-de-valencia/playas?period=2017&period_interval=year)

## :shipit: Does this PR changes any configuration file?
No

### CODE REVIEW

File: config/locales/gobierto_citizens_charters/ca.yml [hunk 1]
```diff
@@ -2,16 +2,28 @@
...
+        about: About this charter
         editions_details: Detall dels compromisos
-        global_progress: Complim els compromisos d'aquesta carta al %{progress}
-        period: 'Datos correspondientes a:'
+        global_progress: Complim els compromisos d'aquesta carta al
+        see_all: Veure la carta completa
+        see_all_subtitle: Missió, Visió, Serveis, Unitat responsable...
     layouts:
       application:
         citizens_charters: Cartas de Serveis
```
Comment (ferblape) [inline config/locales/gobierto_citizens_charters/ca.yml]: This should be "Cartes de servei"

File: config/locales/gobierto_citizens_charters/ca.yml [hunk 2]
```diff
@@ -2,16 +2,28 @@
...
+        global_progress: Compliment global
+        introduction: L'Ajuntament publica les seves cartes de serveis amb els seus
+          respectius compromisos i graus de compliment. Explora les cartes i els nivells
+          de compliment dels compromisos que vam publicar de forma regular
         period: 'Dades corresponents a:'
-        progress: Compliment %{progress}
+        progress: Compliment
+        title: Informació sobre el compliment dels compromisos %{of_the_organization_name}
       show:
+        about: About this charter
```
Comment (ferblape) [inline config/locales/gobierto_citizens_charters/ca.yml]: This should be "Sobre estes cartes"

File: config/locales/gobierto_citizens_charters/ca.yml [hunk 3]
```diff
@@ -2,16 +2,28 @@
...
     charters:
+      details:
+        back: Tornar
       index:
-        charters: Cartes de Serveis
-        commitments: "%{count} compromisos"
-        global_progress: Compliment global %{progress}
+        about: Què són les cartes de serveis
+        commitments:
+          one: 1 compromiso
```
Comment (ferblape) [inline config/locales/gobierto_citizens_charters/ca.yml]: 1 compromís

**Comment by ferblape** [comment]
If I go to http://valencia.gobify.net/cartas-de-servicios/cartas/playas/year/2020 (a year with no data) I think it's confusing to show the page with no data. Maybe it should redirect to the first year with data, which is 2017. What do you think?

**Comment by ferblape** [comment]
For this sparkline:

screen shot 2018-11-05 at 08 53 31

In 2017 the value is 100%, and in 2016 the value is 80%. How can the line be decreasing? Don't should it be in the opposite direction?

**Comment by entantoencuanto** [comment]
I've take a look at the descending sparkline. I've tried to reverse the order the data are presented with the same result. I'll keep research

**Pull request merged**